tests/test_info_schema.py

```diff
@@ -0,0 +1,193 @@
+        from information_schema.columns where table_name = 'EXAMPLE' order by ordinal_position
+        """
+    )
+
+    assert cur.fetchall() == [
+        ("XTIMESTAMP", "TIMESTAMP_NTZ"),
+        ("XTIMESTAMP_NTZ9", "TIMESTAMP_NTZ"),
+        ("XTIMESTAMP_TZ", "TIMESTAMP_TZ"),
+        ("XDATE", "DATE"),
+        ("XTIME", "TIME"),
+        ("XBINARY", "BINARY"),
+        # TODO: support these types https://github.com/tekumara/fakesnow/issues/27
+        # ("XARRAY", "ARRAY"),
+        # ("XOBJECT", "OBJECT"),
+        ("XVARIANT", "VARIANT"),
+    ]
+
+
+def test_info_schema_columns_text(cur: snowflake.connector.cursor.SnowflakeCursor):
+    # see https://docs.snowflake.com/en/sql-reference/data-types-text
+    cur.execute(
+        """
+        create or replace table example (
+            XVARCHAR20 VARCHAR(20), XVARCHAR VARCHAR, XTEXT TEXT
+        )
+        """
+    )
+
+    cur.execute(
+        """
+        select column_name,data_type,character_maximum_length,character_octet_length
+        from information_schema.columns where table_name = 'EXAMPLE' order by ordinal_position
+        """
+    )
+
+    assert cur.fetchall() == [
+        ("XVARCHAR20", "TEXT", 20, 80),
+        ("XVARCHAR", "TEXT", 16777216, 16777216),
+        ("XTEXT", "TEXT", 16777216, 16777216),
+    ]
+
+
+def test_info_schema_databases(conn: snowflake.connector.SnowflakeConnection):
+    # see https://docs.snowflake.com/en/sql-reference/info-schema/databases
+
+    with conn.cursor(snowflake.connector.cursor.DictCursor) as cur:
+        cur.execute("create database db2")
+        cur.execute("select * from information_schema.databases")
+
+        assert cur.fetchall() == [
+            {
+                "database_name": "DB1",
+                "database_owner": "SYSADMIN",
+                "is_transient": "NO",
+                "comment": None,
+                "created": datetime(1970, 1, 1, 0, 0, tzinfo=pytz.utc),
+                "last_altered": datetime(1970, 1, 1, 0, 0, tzinfo=pytz.utc),
+                "retention_time": 1,
+                "type": "STANDARD",
+            },
+            {
+                "database_name": "DB2",
+                "database_owner": "SYSADMIN",
+                "is_transient": "NO",
+                "comment": None,
+                "created": datetime(1970, 1, 1, 0, 0, tzinfo=pytz.utc),
+                "last_altered": datetime(1970, 1, 1, 0, 0, tzinfo=pytz.utc),
+                "retention_time": 1,
+                "type": "STANDARD",
+            },
+        ]
+
+
+def test_info_schema_views_empty(cur: snowflake.connector.cursor.SnowflakeCursor):
+    result = cur.execute("SELECT * FROM information_schema.views")
+    assert result
+    assert result.fetchall() == []
+
+
+def test_info_schema_views_with_views(cur: snowflake.connector.cursor.SnowflakeCursor):
+    cur.execute("CREATE TABLE foo (id INTEGER, name VARCHAR)")
+    cur.execute("CREATE VIEW bar (SELECT * FROM foo WHERE id > 5)")
```

Comment:
to make this work on a real Snowflake instance:
```suggestion
    cur.execute("CREATE VIEW bar AS SELECT * FROM foo WHERE id > 5")
```